src/config/ast.rs

```diff
@@ -89,14 +88,16 @@ impl MatchExpr {
             _ => None,
         };
         for case in &self.cases {
-            if (cfg!(unix) && case.0.exprtype == ExprType::Unix)
-                || os.as_ref().map(|x| *x == case.0.exprtype).unwrap_or(false)
+            if (cfg!(unix) && case.0.exprtype == ExprType::Unix) // is unix
+                || os.as_ref().map(|x| *x == case.0.exprtype).unwrap_or(false) // OS matches the expr
+                || case.0.exprtype == ExprType::Any
```

Comment:
In terms of consistency between the configuration syntax and in the code base, it would be better to rename `ExprType::Any` to `ExprType::Default`.